client/my-sites/site-settings/navigation-back-button/README.md

```diff
@@ -0,0 +1,41 @@
+*Navigation Back Button*
+
+This component is used to display a `Back` button for redirection to the previously
+visited site.
+
+#Usage:
+
+```js
+import NavigationBackButton from 'my-sites/site-settings/navigation-back-button';
+...
+
+getRoute() {
+	... // return a route
+}
+
+<span className="....">
+		<NavigationBackButton redirectRoute={ this.getRoute() } { ...this.props } />
+</span>
+
+```
+
+#Props (required):
+
+- `redirectRoute` -- destination redirect route (string)
+
+##Exemplary use
+
+Redirect to `/settings/manage-connection/:siteSlug`:
+
+```js
+
+getRoute() {
+	const { siteSlug } = this.props;
+
+	if ( siteSlug ) {
+		return '/settings/manage-connection/' + siteSlug;
+	}
+}
+
+```
+- `props` -- used to pass `translate` function to the component
```

Comment:
We usually don't pass `translate` down to other components; we rather prefer wrapping the component that needs to call `translate` with the `localize` HoC. You actually did this already, so you might want to remove this line from the readme.